cluster/mesos/docker/common/bin/await-health-check

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+
+# Copyright 2015 The Kubernetes Authors All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Wait for a service to accept connections.
+# Block up to the timeout duration in seconds (default: 10).
+# Usage: await-health-check [-t=<duration>] <address>
+# Requires: timeout, health-check
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+duration=10
+if [[ "${1:-}" == "-t="* ]]; then
+    duration="${1:3}"
+    [ -z "${duration}" ] && echo "Invalid duration supplied" && exit 1
+    shift
+fi
+
+address=${1:-}
+[ -z "${address}" ] && echo "No address supplied" && exit 1
+
+bin=$(cd $(dirname $0) && pwd -P)
```

Comment:
The scripts in common/bin are added into the docker images, where they're not in the KUBE_ROOT anymore.